app/views/application/index.html.erb

```diff
@@ -1 +1,39 @@
-<h1><%= t('hello') %></h1>
\ No newline at end of file
+<section class="front-section row">
+	<div class="container">
+		<%= image_tag "front-welcome.png", class: "pull-right img-rounded img-front-welcome" %>
+		<h1><%= t "start_page.hello" %></h1>
+		<p class="lead"><%= t "start_page.intro_text" %></p>
+	</div>
+</section>
+
+<section class="front-section row">
+	<div class="container">
+		<h2>Veranstaltungen
+			<a class="btn btn-primary" href="<%= events_path %>">
+				<%= glyph 'calendar' %> <%= t "start_page.all_events" %>
+			</a>
+		</h2>
+		<p class="lead"><%= t "start_page.intro_text_events" %></p>
+
+		<div class="event-list col-md-10 col-md-offset-1">
```

Comment:
In case of responsive design it could be a good idea to add width and offset for smaller devices `col-sm-12 col-md-10 col-sm-offset-0 col-md-offset-1`.